ovs/extensions/services/service.py

```diff
@@ -15,14 +15,20 @@
 """
 Service Factory module
 """
+import time
 
-from subprocess import check_output
+from subprocess import check_output, CalledProcessError
 from ovs.extensions.services.upstart import Upstart
 from ovs.extensions.services.systemd import Systemd
 from ovs.log.logHandler import LogHandler
 
 logger = LogHandler.get('extensions', name='servicemanager')
-
+try:
+    from ovs.extensions.services.fleetctl import FleetCtl
+    HAS_FLEET_CLIENT = True
```

Comment:
No uppercase please for variables that can change
Otherwise all booleans should be uppercase